packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -396,6 +396,17 @@ class ApplicationCommand extends Base {
     }
     return true;
   }
+  
+  /**
+   * When concatenated with a string, this automatically returns the command's mention instead of the ApplicationCommand object.
+   * @returns {string}
+   * @example
+   * // Logs: Command: </ping:123456789012345678>
+   * console.log(`Command: ${command}`);
+   */
+  toString() {
+    return `</${this.name}:${this.id}>`
```

Comment:
If this is to be added, it should use the `chatInputApplicationCommandMention()` formatter from `@discordjs/builders`

Reply:
TypeScript? We are using JavaScript in discord.js and name property declared as a string the the d.ts.

Reply:
My apologies, I wasn't sure if this would error when building/testing.  Will keep that in mind for the future.